packages/webpack-plugin/package.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "@datadog/webpack-plugin",
+    "packageManager": "yarn@4.0.2",
+    "private": true,
+    "license": "MIT",
+    "author": "Datadog",
+    "main": "./dist/packages/webpack-plugin/src/index.js",
+    "exports": {
+        ".": "./src/index.ts",
+        "./*": "./src/*.ts"
+    },
+    "publishConfig": {
+        "access": "public",
+        "exports": {
+            "./package.json": "./package.json",
+            ".": {
+                "types": "./dist/packages/webpack-plugin/src/index.d.ts",
+                "import": "./dist/packages/webpack-plugin/src/index.js",
+                "require": "./dist/packages/webpack-plugin/src/index.js"
+            },
+            "./*": {
+                "types": "./dist/packages/webpack-plugin/src/*.d.ts",
+                "import": "./dist/packages/webpack-plugin/src/*.js",
+                "require": "./dist/packages/webpack-plugin/src/*.js"
+            }
+        }
+    },
+    "files": [
+        "dist/*"
+    ],
+    "scripts": {
+        "build": "yarn clean; tsc",
+        "clean": "rm -rf dist",
+        "prepack": "yarn build",
+        "typecheck": "tsc --noEmit",
+        "watch": "tsc -w"
+    },
+    "devDependencies": {
+        "typescript": "5.4.3"
+    },
+    "dependencies": {
+        "@datadog/build-plugins-core": "workspace:*",
+        "webpack": "5.49.0"
+    },
+    "peerDependencies": {
+        "webpack": "*"
+    },
+    "peerDependenciesMeta": {
+        "webpack": {
+            "optional": true
+        }
+    }
```

Comment:
Is it still optional as thi sis the webpack plugin?

Reply:
Good point, will update.